academy/4-ibc/relayer-intro.md

```diff
@@ -1,38 +1,41 @@
 ---
-title: "Relayers in General"
-order: 8
-description: Relaying in IBC
-tag: deep-dive
+title: "Relaying With IBC"
+order: 3
+description: Introduction to relayers
+tags:
+  - concepts
+  - ibc
+  - dev-ops
 ---
 
-# Relayers in General
+# Relaying With IBC
 
 <HighlightBox type="learning">
 
-In this section, you will learn how relayers fit in IBC.
+In this section, you will learn how relayers fit into the Inter-Blockchain Communication Protocol (IBC).
 
 </HighlightBox>
 
-![IBC overview](/academy/4-ibc/images/ibcoverview.png)
+![IBC overview](/academy/3-ibc/images/ibcoverview.png)
 
-It is useful to briefly recap what relaying is and why it is important. IBC aims to offer blockchains a protocol to enable reliable, secure, and permissionless transfer of packets of data. The protocol is agnostic with respect to the data, paving the way for application developers to develop a range of possible interchain services (fungible and non-fungible token transfers are an obvious candidate, but also arbitrary cross-chain messaging via [interchain accounts](https://interchain-io.medium.com/welcome-to-the-ibc-gang-lets-talk-f469883e0ffe)).
+It is useful to briefly recap what relaying is and why it is important. IBC aims to offer blockchains a protocol to enable reliable, secure, and permissionless transfer of packets of data. The protocol is agnostic concerning the data, paving the way for application developers to develop a range of possible interchain services (fungible and non-fungible token transfers are an obvious candidate, but also arbitrary cross-chain messaging via [interchain accounts](https://interchain-io.medium.com/welcome-to-the-ibc-gang-lets-talk-f469883e0ffe)).
 
-On a high level, this works as follows. A module on a source chain wants to send a packet to a destination chain. It submits a message to the source chain that stores a commitment proof on-chain and logs an event with the packet information. With this information and the proof, you can submit a message to the IBC client on the destination chain, which will verify the proof and (if successful) store a receipt on-chain and have the receiving module execute the required actions according to the packet data. The acknowledgement and timeout functionality has been discussed [previously](./channels.md).
+On a high level, this works as follows. A module on a source chain wants to send a packet to a destination chain. It submits a message to the source chain that stores a commitment proof on-chain and logs an event with the packet information. With this information and the proof, you can submit a message to the IBC client on the destination chain, which will verify the proof and (if successful) store a receipt on-chain and have the receiving module execute the required actions according to the packet data. The acknowledgment and timeout functionality has been discussed [previously](/academy/3-ibc/channels.md).
```

Comment:
```suggestion
On a high level, this works as follows. A module on a source chain wants to send a packet to a destination chain. It submits a message to the source chain that stores a commitment proof on-chain and logs an event with the packet information. With this information and the proof, you can submit a message to the IBC client on the destination chain, which will verify the proof and (if successful) store a receipt on-chain and have the receiving module execute the required actions according to the packet data. The acknowledgement and timeout functionality has been discussed [previously](/academy/3-ibc/channels.md).
```

Reply:
Acknowledgment, unless in code, is always spelled in American English.

Reply:
😱

Reply:
It's a little bit of a hustle for reviews, but trying to keep the code working and the style guide at the same time. The good thing is that most people don't see the e. 😉